src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -6,7 +6,10 @@
                 v-translate="'selectedDataset'"
                 id="selectedDatasetSpan"
             ></span>
-            <a :href="selectedDataset.url" target="_blank">
+            <a v-if="releaseName" :href="selectedDataset.url + '/releases'" target="_blank">
+                {{ selectedDataset.title }} — {{ releaseName }}
```

Comment:
This won't show if the release has since been deleted from adr. The upside is it will update should the release get renamed.

I'm not sure what the desired behaviour is here. If we wanted to uncouple the user's local release from ADR, we could store the release name in Vuex. On the other hand, maybe it is important that the user's release is synchronised with ADR's.